tendrl/ceph_integration/manager/crud.py

```diff
@@ -119,3 +123,14 @@ def delete(self, object_type, object_id):
             )
         else:
             raise NotImplementedError(object_type)
+
+    def sync_request_status(self, request):
+        count = 0
+        while count < 30:
```

Comment:
The request state changes by next hearbeat so I am not sure we actually need lock for pools and rbds. If required can we do in separate PR?